frontend/src/components/Button.cy.tsx

```diff
@@ -0,0 +1,27 @@
+import Button from "./Button";
+
+describe("Button component tests", () => {
+  it("renders", () => {
+    cy.mount(<Button onClick={() => {}}>Click me</Button>);
+    cy.get("button").should("exist");
```

Comment:
when you test our button, better to check text is rendering as well

Reply:
Done